index.html

```diff
@@ -5,184 +5,285 @@
+                    <hr>
+                </div>
+
             </div>
         </div>
-        <button class="my-resume" type="button">Get my resume</button>
-        <ul class="container">
-            <li class="skills">
-                <h5>Languages</h5>
-                <img src="./img/down-arrow.png" alt="">
-            </li>
-            <li class="list list-item icon">
-                <img src="./img/js.png" alt="">
-                <p>JavaScript</p>
-            </li>
-            <li class="list list-item">
-                <img src="./img/html.png" alt="">
-                <p>HTML</p>
-            </li>
-            <li class="list list-item">
-                <img src="./img/css.png" alt="">
-                <p>CSS</p>
-            </li>
-            <li class="skills line">
-                <h5>Frameworks</h5>
-                <img src="./img/right-arrow.png" alt="">
-            </li>
-            <li class="skills line">
-                <h5>Skills</h5>
-                <img src="./img/right-arrow.png" alt="">
-            </li>
-        </ul>
+
     </section>
-    <div class="form-container">
-        <form action="https://formspree.io/f/xyyovkvo" method="post">
-            <h2>Contact me</h2>
-            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-            <input type="text" id="Name" name="Name" placeholder="Your name.." minlength="3" maxlength="30" required>
-            <input type="email" id="email" name="email" placeholder="  Your e-mail.." required>
-            <textarea id="message" name="subject" maxlength="500" placeholder="Write your message here" required></textarea>
-            <input type="submit" value="Get in touch">
-        </form>
-        <div class="endline">
+    <section id="contact" class="form-container">
+        <div class="form-min">
+            <div class="form-des">
+                <h1>Contact me</h1>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            </div>
+            <form method="POST" action="https://formspree.io/f/xyyovkvo">
+                <ul class="form-ul">
+                    <li>
+                        <input type="text" name="name" id="user-name" placeholder="Your name here.." required>
+                    </li>
+                    <li>
+                        <input type="email" name="email" id="email-address" placeholder="Your E-mail here.." required>
+                    </li>
+                    <li>
+                        <textarea name="message" id="msg" maxlength="500" placeholder="Write your Message Here" required></textarea>
+                    </li>
+                    <li>
+                        <button type="submit">Get In Touch</button>
+                    </li>
+                </ul>
```

Comment:
- [ ] Make sure your **inputs text fields** design matches correctly the one from Figma design. Kindly use the correct **paddings(left, top, bottom & right)** on inputs text, **text color**, **border-radius**,.... to make your project sticks to the Figma design.
#### Figma design
![f1](https://user-images.githubusercontent.com/83042742/173172387-a6704474-573a-4f23-9f79-f0d6ccea5384.png)

#### Your design
![f2](https://user-images.githubusercontent.com/83042742/173172410-dd3b0aa7-adb4-435e-a4b4-2dab7d1d8a6f.png)